proto/messages.proto

```diff
@@ -159,6 +159,17 @@ message Agent {
   string deployment_group_id = 4;
 }
 
+message AppDeploymentStatus {
+  string id = 1;
+  string deployment_group_id = 2;
+  string version = 3;
+  string status = 4;
```

Comment:
Do we want to use a Enum for the status and explicitly list the possible values? /cc @jonatanklosko